src/App.js

```diff
@@ -1,19 +1,42 @@
 import React from 'react';
-
-import './App.scss';
+import { Tabs } from './Tabs';
+// import './App.scss';
 
 /* eslint-disable */
-const tabs = [
+const tabsFromServer = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    tabs: tabsFromServer.map(tab => ({ ...tab })),
+    selectedTabId: 'Tab 1',
```

Comment:
краще видягувати з масиву табу, яка має бути активною
```suggestion
    selectedTabId: tabsFromServer[0].id,
```